_api-reference/cat/cat-aliases.md

```diff
@@ -10,12 +10,12 @@ has_children: false
 ---
 
 # CAT aliases
-Introduced 1.0
+**Introduced 1.0**
 {: .label .label-purple }
 
 The CAT aliases operation lists the mapping of aliases to indexes, plus routing and filtering information.
 
-## Example
+**Example**
```

Comment:
According to the API template, we should move Example request and Example response to H4-level sections under `## Request fields`.

However, I think it would make more sense to add each example as H2 or H3 level headers under their respective field sections. @kolchfa-aws, @hdhalter, @vagimeli, any thoughts on this?

Reply:
In the case of multiple examples, we can provide several H4 level headers with descriptions (see http://opensearch.org/docs/latest/api-reference/index-apis/stats/).

Reply:
Ok noted @Naarcha-AWS. I will do the changes.